modules/nodes-sigstore-configure-cluster-policy.adoc

```diff
@@ -6,14 +6,19 @@
 [id="nodes-sigstore-configure-cluster-policy_{context}"]
 = Creating a cluster image policy CR
 
+// Abstract from https://github.com/openshift/openshift-docs/pull/81985/files#diff-1f70afdcabd5c3ddafa70557887dc5dc5d5b00705e9f67b20ca4d2e99b17c8efR48
+
+[role="_abstract"]
+Cluster administrators can use a `ClusterImagePolicy` custom resource (CR) to declare an image policy for the cluster, requiring signature validation and other policy checks in addition to image integrity.
+
 A `ClusterImagePolicy` custom resource (CR) enables a cluster administrator to configure a sigstore signature verification policy for the entire cluster. When enabled, the Machine Config Operator (MCO) watches the `ClusterImagePolicy` object and updates the `/etc/containers/policy.json` and `/etc/containers/registries.d/sigstore-registries.yaml` files on all the nodes in the cluster.
 
 The following example shows general guidelines on how to configure a `ClusterImagePolicy` object. For more details on the parameters, see "About cluster and image policy parameters."
 
-The default `openshift` cluster image policy provides sigstore support for the required {product-title} images. This cluster image policy is active only in clusters that have enabled Technology Preview features. You must not remove or modify this cluster image policy object. Cluster image policy names beginning with `openshift` are reserved for future system use.
-
-:FeatureName: The default `openshift` cluster image policy
-include::snippets/technology-preview.adoc[]
+[NOTE]
+====
+The default `openshift` cluster image policy provides sigstore support for the required {product-title} images, which are stored in the `quay.io/openshift-release-dev/ocp-release` repository. You must not remove or modify this cluster image policy object. Cluster image policy names beginning with `openshift` are reserved for future system use.
```

Comment:
```suggestion
The default `openshift` ClusterImagePolicy object provides sigstore support for the required {product-title} images, which are stored in the `quay.io/openshift-release-dev/ocp-release` repository. You must not remove or modify this cluster image policy object. Cluster image policy names beginning with `openshift` are reserved for future system use.
```

I just saw some upstream docs refer to API objects by their `Kind` , for explame https://kubernetes.io/docs/tasks/administer-cluster/quota-api-object/  uses `PersistentVolumeClaim`, `ResourceQuota`